pkg/addon-operator/operator.go

```diff
@@ -2112,3 +2112,16 @@ func (op *AddonOperator) taskPhase(tsk sh_task.Task) string {
 	}
 	return ""
 }
+
+// OnFirstConvergeDone run addon-operator business logic when first converge is done and operator is ready
+func (op *AddonOperator) OnFirstConvergeDone() {
+	go func() {
+		<-op.ConvergeState.firstRunDoneC
+		time.Sleep(1 * time.Second) // wait for service to be resolved
```

Comment:
```suggestion
		time.Sleep(1 * time.Second) // wait for service to be resolved
```
Could you elaborate? I didn't get this.

Reply:
Add comment